Resources/Prototypes/Entities/Objects/Misc/implanters.yml

```diff
@@ -139,6 +140,7 @@
   components:
     - type: Implanter
       implant: LightImplant
+      allowMultipleImplants: true # ok this might be a bad idea. How to win against your enemies? Crash their game with 50 lights!
```

Comment:
These could all just be false, tbh. I dont really see a point in allowing multiple.

Reply:
it's VERY funny to have 10 honk implants